### PR TITLE
Improvements on GCE infra #863, instance group for gcleanup

### PR DESCRIPTION
**Pull request by DamianSzymanski**
## What is the problem that this PR is trying to fix?

With gcloud cleanup sitting on heroku we don't have access to redis on gpc, we want to move service to the instance on GPC

## What approach did you choose and why?

Build and deploy service to the instance group on GPC

## What feedback would you like, if any?

any welcome

### CODE REVIEW

File: modules/gcloud_cleanup/variables.tf [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+variable "gcloud_cleanup_scale" {
+  default = "worker=1:Standard-1X"
+}
+
+variable "gcloud_cleanup_version" {
+  default = "master"
+}
+
+variable "gcloud_cleanup_docker_self_image" {
+  default = "travisci/gcloud-cleanup:latest"
```
Comment (soulshake) [inline modules/gcloud_cleanup/variables.tf]: As far as I know, we're not yet making use of the `latest` tag, so [`travisci/gcloud-cleanup:latest` on Docker Hub](https://hub.docker.com/r/travisci/gcloud-cleanup/tags) is 6 months old.

File: modules/gcloud_cleanup/variables.tf [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+variable "gcloud_cleanup_opencensus_tracing_enabled" {
+  default = "false"
+}
+
+variable "gcloud_cleanup_scale" {
+  default = "worker=1:Standard-1X"
+}
+
+variable "gcloud_cleanup_version" {
```
Comment (soulshake) [inline modules/gcloud_cleanup/variables.tf]: Where is this used?

Comment (DamianSzymanski) [reply modules/gcloud_cleanup/variables.tf]: It's not used, will remove

File: modules/gcloud_cleanup/variables.tf (hunk 1 above)
Comment (DamianSzymanski) [reply modules/gcloud_cleanup/variables.tf]: Not sure what to put here, I used `b28c85d` for staging build, but I guess we want to use latest  here. We could push to latest after this merge https://github.com/travis-ci/gcloud-cleanup/pull/53 wdyt?

**Comment by Duologic** [comment]
@DamianSzymanski Is this still relevant if we are running on GKE?

**Pull request closed without merging**